### DISCUSSION TITLE
Icon slide animation inside the button

### DISCUSSION THREAD

**Discussion by naveensunkara** [sButtons/sbuttons#1470] [Ideas]
This is a suggestion I thought it might be a good option. There are lot of scenarios where the buttons will have an icon. Simple animations on the icon will enhance the UX. I just tried one of the effect. Looking forward for feedback.

https://pchfx.csb.app/

**Comment by shahednasser** [comment]
Hi @naveensunkara that looks really nice. I'll create an issue for this discussion and you can add these new button classes.
